package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "publish",
+  "version": "0.1.0-dev",
```

Comment:
1.0.0 seems reasonable to me, in line with our other thinking - since I believe "If used to publish itself", the string "-dev" will be added automatically?